grails-app/services/au/org/emii/gogoduck/job/JobExecutorService.groovy

```diff
@@ -1,11 +1,82 @@
 package au.org.emii.gogoduck.job
 
+import au.org.emii.gogoduck.worker.Worker
+
+import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.BlockingQueue
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
 class JobExecutorService {
 
+    static BlockingQueue<Map> JOB_QUEUE
```

Comment:
I'd say then lets have a limit here. Unlikely that it'll happen, but heaps more robust. Will it be too much effort?